deploy/kubernetes/examples/demo-pvc-clone-pvc.yaml

```diff
@@ -0,0 +1,15 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: demo-pvc-clone-pvc
```

Comment:
why _clone_ and not _cloned_?
`demo-pvc-cloned-pvc`

Reply:
no reason, the operation is "clone pvc", should I change?

Reply:
IMO, yes

Reply:
done

Reply:
also update the file name accordingly

Reply:
btw, fyi, "clone" is also a noun

Reply:
done